apps/system-e2e/src/tests/judicial-system/smoke/shared-steps/send-appeal.ts

```diff
@@ -51,12 +72,34 @@ export async function prosecutorAppealsCaseTest(page: Page, caseId: string) {
   const statementFileChooser = await statementFileChooserPromise
   await page.waitForTimeout(1000)
   await statementFileChooser.setFiles(
-    await createFakePdf('TestGreinargerd.pdf'),
+    await createFakePdf('TestGreinargerdSaekjanda.pdf'),
   )
   await Promise.all([
     verifyRequestCompletion(page, '/api/graphql', 'CreatePresignedPost'),
     verifyRequestCompletion(page, '/api/graphql', 'CreateFile'),
   ])
+
+  const statementCaseFileChooserPromise = page.waitForEvent('filechooser')
+  await page
+    .locator('section')
+    .filter({
+      hasText:
+        'GögnEf ný gögn eiga að fylgja greinargerðinni er hægt að hlaða þeim upp hér að n',
+    })
+    .locator('button')
+    .click()
+
+  const statementCaseFileChooser = await statementCaseFileChooserPromise
+  await page.waitForTimeout(1000)
```

Comment:
Why not wait for some _thing_ instead of some _time_?

Reply:
We actually found a bug in our code related to how we update our cases thanks to the tests (since they run so quickly) that we're fixing as we speak, and that's going to make this irrelevant so we'll remove it when we've finished that :D